rend3/src/managers/mesh.rs

```diff
@@ -59,6 +65,8 @@ impl MeshBuffers {
         rpass.set_vertex_buffer(3, self.vertex_uv0.slice(..));
         rpass.set_vertex_buffer(4, self.vertex_uv1.slice(..));
         rpass.set_vertex_buffer(5, self.vertex_color.slice(..));
+        rpass.set_vertex_buffer(6, self.vertex_joint_index.slice(..));
+        rpass.set_vertex_buffer(7, self.vertex_joint_weight.slice(..));
```

Comment:
There's a really subtle problem here, gpu mode adds a vertex attribute at the very end, so we need to go bump the index on that to 8.